Backend/controllers/PatientsController.js

```diff
@@ -123,7 +151,25 @@ exports.update_patient = async (req, res) => {
                 status:oldData.status,
             })
             patient.save();
+
+
+            //----- updating log ----//
+            let date = new Date();
+            date.setHours(0,0,0,0);
+
+            let log =await PatientLog.findOne({date:date,test_status:patient.status});
+
+            if(log){
+                log.count+=1;
+                await log.save();
+            }else{
+                await new PatientLog({test_status:patient.status, date: date}).save();
```

Comment:
This does mean that a certain day patient log won't be created if no patients were tested on that day, right? Just checking